src/pushsource/_impl/backend/koji_source.py

```diff
@@ -504,6 +514,108 @@ def _push_items_from_container_build(self, nvr, meta):
 
         return out
 
+    def _push_items_from_vmi_build(self, nvr, meta):
+        LOG.debug("Looking for virtual machine images on %s, %s", nvr, meta)
+
+        if not meta:
+            message = "Virtual machine image build not found in koji: %s" % nvr
+            LOG.debug(message)
+            raise ValueError(message)
+
+        if meta.get("container_koji_task_id"):
```

Comment:
container_koji_task_id is in extra :-)
* https://github.com/containerbuildsystem/atomic-reactor/blob/master/docs/koji.md?plain=1#L79

Sorry I wrote wrong example in the other comment

Reply:
IMHO we can even  add link to the OSBS documentation for container_koji_task_id as a comment.

Reply:
Comment added